tests/automatic_tags_der.rs

```diff
@@ -0,0 +1,42 @@
+pub use asn1rs::prelude::*;
+
+asn_to_rust!(
+    r"Data-Structures DEFINITIONS AUTOMATIC TAGS ::=
+BEGIN
+  DataStructures ::= SEQUENCE
+  {
+     int              INTEGER,
+     limitedString    UTF8String (SIZE(1..16)),
+     optionalString   UTF8String OPTIONAL,
+     enumerated       ENUMERATED {value1, value2, value3},
+     optionalChoice   CHOICE
+     {
+        int1   INTEGER,
+        int2   INTEGER
+     }  OPTIONAL,
+     sequenceOfString SEQUENCE OF UTF8String
+  }
+END"
+);
+
+#[test]
+fn automatic_tags_der() {
+    let der_content = b"0'\x80\x0209\x81\tSomething\x83\x01\x00\xa4\x05\x81\x03\x00\xd41\xa5\x0c\x0c\x04abcd\x0c\x04efgh".to_vec();
```

Comment:
Thanks for changing the example. In the long term, I prefer this to have the same notation as in the [remaining crate](https://github.com/kellerkindt/asn1rs/blob/master/tests/basic_ia5string.rs#L36)

Reply:
Sure. After implementing `Write`, I'll write separate tests for each data type like existing tests